docker/php.Dockerfile

```diff
@@ -220,14 +222,15 @@ RUN wget -O /usr/local/bin/php-fpm-healthcheck https://raw.githubusercontent.com
 #============================================
 # FPM Extensions
 #============================================
-RUN docker-php-ext-enable dom && \
-    docker-php-ext-enable gd && \
-    docker-php-ext-enable igbinary && \
+RUN docker-php-ext-enable gd && \
     docker-php-ext-enable opcache && \
     docker-php-ext-enable pdo_pgsql && \
     docker-php-ext-enable redis && \
-    docker-php-ext-enable simplexml && \
-    docker-php-ext-enable zip
+    docker-php-ext-enable simplexml
+RUN rm /usr/local/etc/php/conf.d/docker-php-ext-igbinary.ini && \
+    rm /usr/local/etc/php/conf.d/docker-php-ext-pcntl.ini && \
+    rm /usr/local/etc/php/conf.d/docker-php-ext-sockets.ini && \
+    rm /usr/local/etc/php/conf.d/docker-php-ext-zip.ini
```

Comment:
Are we installing the extensions but not enabling them? Is this why you're deleting their settings here?

Reply:
pcntl/sockets/zip are bundled extensions, thus the build happens in the shared build stage, we're disabling them as we do not use them in `php-fpm` :-)